packages/web-app/hooks/useCohort.tsx

```diff
@@ -0,0 +1,174 @@
+import { SafeTransaction } from "@safe-global/safe-core-sdk-types";
+import Safe from "@safe-global/safe-core-sdk";
+import SafeServiceClient, {
+	SafeMultisigConfirmationResponse,
+} from "@safe-global/safe-service-client";
+import EthersAdapter from "@safe-global/safe-ethers-lib";
+import { arbitratorAddress, cohortAddress, safeTxServiceUrl } from "../lib/constants";
+import { useEffect, useState } from "react";
+import { ethers, Signer, BigNumber, constants } from "ethers";
+import { useAccount, useSigner } from "wagmi";
+import { ResolutionInput } from "./useArbitrator";
+
+export type ResolutionProposal = {
+	txHash: string;
+	txNonce: number;
+	confirmationsRequired: number;
+	confirmations: SafeMultisigConfirmationResponse[];
+	resolution: ResolutionInput;
+};
+
+const useSafe = ({
```

Comment:
May we create separate files for each Hook to maintain coherence in naming?

Reply:
Agreed, you can do it